lib/services/platform-service.ts

```diff
@@ -780,7 +780,9 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		const platformData = this.$platformsData.getPlatformData(platform, projectData);
 		const prepareInfo = this.$projectChangesService.getPrepareInfo(platform, projectData);
 		// In case when no platform is added and webpack plugin is started it produces files in platforms folder. In this case {N} CLI needs to add platform and keeps the already produced files from webpack
-		if (appFilesUpdaterOptions.bundle && this.isPlatformInstalled(platform, projectData) && !this.$fs.exists(platformData.configurationFilePath) && (!prepareInfo || !prepareInfo.nativePlatformStatus || prepareInfo.nativePlatformStatus !== constants.NativePlatformStatus.alreadyPrepared)) {
+		if (appFilesUpdaterOptions.bundle && this.isPlatformInstalled(platform, projectData)
+			&& !this.$fs.exists(platformData.configurationFilePath)
+			&& (!prepareInfo || !prepareInfo.nativePlatformStatus || prepareInfo.nativePlatformStatus !== constants.NativePlatformStatus.requiresPlatformAdd)) {
```

Comment:
shouldn't this be `prepareInfo.nativePlatformStatus === constants.NativePlatformStatus.requiresPlatformAdd`

Reply:
No. In case when no platform is added and the user executes cloud build, {N} CLI will produce `.nsprepareinfo` file with the following content:
```
{
    nativePlatformStatus: "1"
}
```

If the cloud build is ran for second time, {N} CLI should'n add the platform again so the check should be
```
prepareInfo.nativePlatformStatus !== constants.NativePlatformStatus.requiresPlatformAdd
```

Reply:
Okay, I agree with the described behavior, but what will happen in case the `nativePlatformStatus` has value "2" - does it mean we have to call `addPlatform` again?